### ISSUE TITLE
Whitespace not preserved between source and output

### ISSUE THREAD

**Issue by crclark96** [theryangeary/choose#31]
This ruins pre-formatted tables and requires extra chaining to return the output to its original state, meaning `choose` is not a transparent translation utility as much as a complete refactoring.

Examples:

```
➜ sdiff <(choose 0:4 -i mnemonics.tmp | head ) <(head mnemonics.tmp )
0x2813: 0x0a, 0, 2, 2                                         | 0x2813: 0x0a,  0,   2,            2
0xb537: 0x2d, 6, 17, 34                                       | 0xb537: 0x2d,  6,  17,           34
0x460c: 0x11, 2, 5, 170                                       | 0x460c: 0x11,  2,   5,          170
0x4615: 0x11, 2, 5, 850                                       | 0x4615: 0x11,  2,   5,          850
0xd814: 0x36, 9, 29, 24650                                    | 0xd814: 0x36,  9,  29,        24650
0xa901: 0x2a, 5, 13, 320450                                   | 0xa901: 0x2a,  5,  13,       320450
0xf7f4: 0x3d, 11, 37, 11856650                                | 0xf7f4: 0x3d, 11,  37,     11856650
0x2800: 0x0a, 0, 2, 23713300                                  | 0x2800: 0x0a,  0,   2,     23713300
0xda06: 0x36, 9, 29, 687685700                                | 0xda06: 0x36,  9,  29,    687685700
0x4244: 0x10, 1, 3, 2063057100                                | 0x4244: 0x10,  1,   3,   2063057100
```

even with `columns -t` it is slightly off in terms of justification:

```
➜ sdiff <(choose 0:4 -i mnemonics.tmp | head | column -t) <(head mnemonics.tmp )
0x2813:  0x0a,  0,   2,   2                                   | 0x2813: 0x0a,  0,   2,            2
0xb537:  0x2d,  6,   17,  34                                  | 0xb537: 0x2d,  6,  17,           34
0x460c:  0x11,  2,   5,   170                                 | 0x460c: 0x11,  2,   5,          170
0x4615:  0x11,  2,   5,   850                                 | 0x4615: 0x11,  2,   5,          850
0xd814:  0x36,  9,   29,  24650                               | 0xd814: 0x36,  9,  29,        24650
0xa901:  0x2a,  5,   13,  320450                              | 0xa901: 0x2a,  5,  13,       320450
0xf7f4:  0x3d,  11,  37,  11856650                            | 0xf7f4: 0x3d, 11,  37,     11856650
0x2800:  0x0a,  0,   2,   23713300                            | 0x2800: 0x0a,  0,   2,     23713300
0xda06:  0x36,  9,   29,  687685700                           | 0xda06: 0x36,  9,  29,    687685700
0x4244:  0x10,  1,   3,   2063057100                          | 0x4244: 0x10,  1,   3,   2063057100
```

reformatting is technically possible with `column -t -R 3,4,5`, but `-R` is unsupported on default builds of `column`. Additionally, this requires specifying all column numbers to be right-justified, which is infeasible for files with large numbers of right-justified columns

```
➜ choose --version
choose 1.3.0
```

**Comment by theryangeary** [comment]
`choose` does not preserve whitespace when whitespace is the field separator (as is the default). With whitespace as the field separator, it would be impossible to make a decision as to what field the white space belongs in a meaningful way. Instead, if you require whitespace preservation, consider using the `--field-separator` option to use a different field separator, so that white space will be included in the fields. Alternatively, consider using the `--character-wise` option.

In this case either of the following would achieve your goal:

```
choose -f ',' -o ',' -i mnemonics.tmp 0:4
choose -c  -i mnemonics.tmp 0:34
```

**Comment by theryangeary** [comment]
Do note that due to a bug in v1.3.0, an extra newline was erroneously printed with that first command. It has been fixed in v1.3.1.
